dynawo/sources/Modeler/ModelManager/DYNModelManager.cpp

```diff
@@ -473,6 +476,7 @@ ModelManager::initParams() {
   associateBuffers();
   // call the parameter calculation method
   solveParameters();
+  checkDataCoherence(getCurrentTime());
```

Comment:
You want to do it specifically on the initialization model there and not the dynamic one?

Reply:
Precisely. It is already done on the dynamic model later.
If we do not add this test, then with the aliases some checks are ignored as they are only available in the initialization model (whereas before it was also done in the dynamic one)